### DISCUSSION TITLE
Using exogenous variables with crossvalidation in mlforecast

### DISCUSSION THREAD

**Discussion by mkrech** [Nixtla/mlforecast#325] [Q&A]
Hi,
thank you for your fantastic library.

My question:
Is there example or a way to combine exogenous variables eg fourier (future_df, transformed_df) to combine with crossvalidation.

I found backtestsplits in utilsforecast. I could be done "by hand" :). Is there another way?

regards
Michael

**Comment by jmoralez** [comment]
Hey @mkrech, thanks for using mlforecast. You can merge the transformed_df onto your dataframe and set `static_features` to a list with your static features (or`[]` if you don't have any) and mlforecast will automatically assign the future exogenous features during cross validation.
